### ISSUE TITLE
Relative Due Triggers

### ISSUE THREAD

**Issue by algeorg** [GothenburgBitFactory/taskwarrior#2096]
#### To request a feature...

* Clearly describe the feature.
Being able to set relative due/wait/etc dates that trigger upon being unblocked and possibly other events. 

* Clearly state the use case. We are only interested in use cases, do not waste time with implementation details or suggested syntax.

I use a GTD methodology and make a lot of projects that utilize many dependencies. Many of my projects involve a series of steps I need to do and at some point waiting for somebody to get back to me. I tend to set waiting to something like a few days, after which I'd like said task to end up in my task next so that I can be reminded that I need to followup with said person and see what's going on on their end. The issue is that if I set that specific task to `wait:1w`, it's 1week from the day I made it, not 1 week from the date it is ready and no longer blocked by dependencies. The same issue can be said for recurrances and I imagine anything else.

* Please see our notes on [How to request a feature](https://taskwarrior.org/docs/features.html)

**Comment by JonasDralle** [comment]
This feature has already been specified in the [Recurrence RFC](https://taskwarrior.org/docs/design/recurrence.html) (but as not implemented yet).

**Comment by pbeckingham** [comment]
This is a deferred evaluation feature we intend to add.

**Comment by Xerus2000** [comment]
yes, that recurrence RFC sounds great - having regular tasks such as mowing the lawn or cutting nails or whatever really need the chain feature, exactly what I was searching for - plans for implementation?

**Comment by matmilbury** [comment]
Will this relative recurrence ever be implemented? Is that tracked somewhere?

If I understand correctly, events would reoccur 1m after being marked as done, not every month from the first occurrence.

Also, the Recurrence RFC link is dead.

**Comment by djmitche** [comment]
I think the RFC is now at https://github.com/GothenburgBitFactory/taskwarrior/blob/develop/doc/devel/rfcs/recurrence.md

I don't know of anyone working on this right now.

**Comment by matmilbury** [comment]
Cool, thank you @djmitche for an updated link.

This rfcs looks awesome :fire:
